packages/file/src/index.coffee.md

```diff
@@ -224,7 +224,7 @@ require('nikita')
           Replace this marker, default to the replaced string if missing.
           """
         'mode':
-          type: 'integer'
+          oneOf: [{type: 'string'}, {type: 'number'}]
```

Comment:
JSON schema offers a distinction between `integer` and `number`, why using number. Also, you shall at least be consistent with the rest of the code. Either use `integer` like everywhere else or fix it everywhere with an explanation (and a test case reproducing the error). You will also notice the description to be much more detailed now. Finally, you shall probably not even declare it literraly but instead use a `$ref` and point it to `fs.base.chmod`